### ISSUE TITLE
Error when running create_urbancars.sh

### ISSUE THREAD

**Issue by HowToNameMe** [facebookresearch/Whac-A-Mole#4]
We download the provided ckpt via the following code
```
wget https://dl.fbaipublicfiles.com/maskformer/panoptic-coco/maskformer_panoptic_swin_large_IN21k_384_bs64_554k/model_final_7505c4.pkl
```
However, we find that the model cannot segment all the images successfully. Thus, some images have no corresponding mask after conducting segmentation, which leads to the error in generating urbancars dataset with following command
```
PYTHONPATH=.:$PYTHONPATH python create_datasets/urbancars/gen_urbancars.py --split train
PYTHONPATH=.:$PYTHONPATH python create_datasets/urbancars/gen_urbancars.py --split val
PYTHONPATH=.:$PYTHONPATH python create_datasets/urbancars/gen_urbancars.py --split test
```
So we want to know how to deal with **the images without corresponding masks**. Should we just neglect all of them, or we can tackle this problem with other methods.

**Comment by zhihengli-UR** [comment]
Hi!

I appreciate your interest in our work!

I did not encounter this issue when I was creating the UrbanCars dataset. Can you confirm that you used the official Stanford Cars Dataset downloaded from [the official website](https://ai.stanford.edu/~jkrause/cars/car_dataset.html)? Unfortunately, the link to the official website for the Stanford Cars dataset is broken.

**Comment by theamaya** [comment]
@HowToNameMe did you find a workaround for this? I am encountering the same issue.

**Comment by HowToNameMe** [comment]
@zhihengli-UR No. The original link is broken. So, we download stanford_car dataset shared by other researchers. Could you please share the standford_car dataset you keep to us via Google Drive or any possible way. We will be more than grateful. Thanks!

**Comment by HowToNameMe** [comment]
@theamaya Not yet.
